templates/WG-README.md

```diff
@@ -0,0 +1,83 @@
+# CHAOSS \<_wg-name_\> Working Group
```

Comment:
```suggestion
# CHAOSS <wg-name> Working Group
```

Reply:
Backslashes are required else the word gets omitted, removed underscores